utils/wallet/provider/uphold/uphold.go

```diff
@@ -55,8 +55,12 @@ const (
 )
 
 const (
-	sandboxFingerprint = "d961PqM/qoU9+Oc1t0v9V76A9ZlB5Dx/RFnhi4EkLIA="
-	prodFingerprint    = "d961PqM/qoU9+Oc1t0v9V76A9ZlB5Dx/RFnhi4EkLIA="
+	// The Leaf Certificates
+	//sandboxFingerprint = "d961PqM/qoU9+Oc1t0v9V76A9ZlB5Dx/RFnhi4EkLIA="
```

Comment:
We can just delete these, if we need them again we can lookup or grab from git history